### ISSUE TITLE
Create Dataset from multiple files

### ISSUE THREAD

**Issue by cswangjiawei** [pytorch/text#527]
I noticed that the 'path' parameter of the splits method of the Dataset class accepts only a single file path . If the data consists of multiple files, such as the train set and test set of 1 billion word, which are composed of multiple files, how to create the Dataset object?
